locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/AbstractComputeServiceRegistry.java

```diff
@@ -57,7 +57,7 @@
     private final Map<Map<?, ?>, ComputeService> cachedComputeServices = new ConcurrentHashMap<>();
 
     @Override
-    public ComputeService findComputeService(ConfigBag conf, boolean allowReuse) {
+    public ComputeService findComputeService(ConfigBag conf, boolean allowReuse) throws IllegalArgumentException {
```

Comment:
This method override the one from the super class, the signature shouldn't change. Actually, it doesn't need to as `IllegalArgumentException` is a runtime exception